DEV_GUIDE.md

```diff
@@ -117,23 +117,15 @@ No one likes to argue about code formatting in pull requests, as project we take
 Build with the `dist` profile as shown above, then execute this:
 
 ```shell
-$ java -jar kroxylicious-app/target/kroxylicious-app-*-SNAPSHOT.jar --config {path-to-kroxylicious-config}
+$ kroxylicious-app/target/kroxylicious-app-*-bin/kroxylicious-app-*/bin/kroxylicious-start.sh --config {path-to-kroxylicious-config}
```

Comment:
We've got a bit of inconsistency with placeholders in the text.  We've got curly brackets here and dollars used down below.  We should use a single style.